imagetest/cmd/manager/main.go

```diff
@@ -159,8 +159,10 @@ func main() {
 	}
 
 	out, err := imagetest.RunTests(ctx, testWorkflows, *outPath, *project, *zone, *parallelCount)
+	if out != nil {
```

Comment:
no we want this how we had it, this failure means a failure to run the tests. there's an idiomatic pattern in golang which is that the non-error output is not to be trusted or used when err is not nil.